frontend-test.sh

```diff
@@ -0,0 +1,12 @@
+COMPOSE_FILE_LOC=docker/docker-compose.test.yml
```

Comment:
```suggestion
#!/bin/bash

COMPOSE_FILE_LOC=docker/docker-compose.test.yml
```

having a shebang for scripts is generally a good idea: https://en.wikipedia.org/wiki/Shebang_(Unix)